code/game/objects/items/robot/items/rlf.dm

```diff
@@ -0,0 +1,50 @@
+/obj/item/rlf
```

Comment:
Not sure if this typepath is really descriptive, can you make it a tad bigger? Something like
`/obj/item/rapid_lollipop_fabricator`